share/share.go

```diff
@@ -61,7 +61,7 @@ func (s *ShareWithProof) Validate(rootHash []byte, x, y, edsSize int) bool {
 		namespace = GetNamespace(s.Share)
 	}
 	return s.Proof.VerifyInclusion(
-		sha256.New(), // TODO(@Wondertan): This should be defined somewhere globally
+		sha256.New(),
```

Comment:
Its still is though

Reply:
This happens in many places, so worth to have an issue instead of TODO